wrappers/python/src/cxx/api/ExportObservables.h

```diff
@@ -63,7 +59,6 @@ using obs_handle_t = readdy::ObservableHandle;
 
 inline obs_handle_t registerObservable_Reactions(sim &self, unsigned int stride,
                                                  const py::object& callback = py::none()) {
-    self.context().recordReactionsWithPositions() = true;
```

Comment:
is it safe to remove this at this point?

Reply:
Currently if the flag is not set, it will be set later in `Reactions::initialize`, where also the warning is printed. I found it weird that the python-binding is responsible for setting this flag, when the observable can do this itself.

Reply:
I agree the binding is probably not the right place. I would put it into the Simulation implementation of registering a reaction observable and not just remove it. The warning is only printed if the flag wasn't already switched on which can be seen as an invalid state for the context should such an observable be registered.